CHANGELOG.md

```diff
@@ -10,6 +10,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Breaking Changes
 - Modify `Signer`'s `signTransaction` method to return an array of signed bytes rather than a string based signature.
 
+### Added gRPC-Web
+The release switches the library over to using gRPC-Web instead of the Node.js version of gRPC. The goal of this change is to enable the library to be usable on the web and Node.js without having two separate codebases or build strategies.
+
+- Moved from gRPC node to gRPC-Web
+- Updated ESLint and configured it to work with prettier and TSConfig.
+- Fixed the lint issues found in the src as well as the tests.
+- Moved the generated files that are being consumed by the build process into the src.
+- Updated webpack configuration to use UMD (Universal module definitions) to support both web and Node.js.
```

Comment:
I think these can be broken up a little more.

Some of these are only internal, like updating ESLint and fixing lint issues, so could go under an `### Internal` or `### Other` heading.

I'm not sure if moving the generated files is internal or not, so not sure where to put that.

I'm also not sure I would put `gRPC-Web` directly into the `### Added` header.

This is the changelog format we're trying to be based off: https://keepachangelog.com/en/1.0.0/

I also really like the Mocha changelog (although it deviates from that format a little):
https://github.com/mochajs/mocha/blob/master/CHANGELOG.md